pkg/work/spoke/controllers/manifestcontroller/manifestwork_controller.go

```diff
@@ -175,6 +175,9 @@ func (m *ManifestWorkController) sync(ctx context.Context, controllerContext fac
 	// if needed.
 	if !mwUpdated && !amwUpdated && requeueTime < MaxRequeueDuration {
 		controllerContext.Queue().AddAfter(manifestWorkName, requeueTime)
+	} else {
+		// resync each manifestwork every 5 minutes
+		controllerContext.Queue().AddAfter(manifestWorkName, 5*time.Minute)
```

Comment:
I do not think we need else here, we do not need to requeue when manifestwork or appliedmanfiestwork is udpated. I think we only need to change this line
if !mwUpdated && !amwUpdated && requeueTime < MaxRequeueDuration
to 
if !mwUpdated && !amwUpdated

Reply:
Updated as suggested.